docs/content/preview/releases/ybdb-releases/v2.25.md

```diff
@@ -0,0 +1,311 @@
+linkTitle: v2.25 series (Preview)
+description: Enhancements, changes, and resolved issues in the v2.25 preview release series.
+menu:
+  preview_releases:
+    identifier: v2.25
+    parent: ybdb-releases
+    weight: 2790
+type: docs
+---
+
+What follows are the release notes for the YugabyteDB v2.25 release series. Content will be added as new notable features and changes are available in the patch releases of the YugabyteDB v2.25 release series.
+
+For an RSS feed of all release series, point your feed reader to the [RSS feed for releases](../index.xml).
+
+## v2.25.0.0 - January 14, 2025 {#v2.25.0.0}
+
+**Build:** `2.25.0.0-b489`
+
+### Downloads
+
+<ul class="nav yb-pills">
+  <li>
+    <a href="https://downloads.yugabyte.com/releases/2.25.0.0/yugabyte-2.25.0.0-b489-darwin-x86_64.tar.gz">
+      <i class="fa-brands fa-apple"></i>
+      <span>macOS</span>
+    </a>
+  </li>
+  <li>
+    <a href="https://downloads.yugabyte.com/releases/2.25.0.0/yugabyte-2.25.0.0-b489-linux-x86_64.tar.gz">
+      <i class="fa-brands fa-linux"></i>
+      <span>Linux x86</span>
+    </a>
+  </li>
+  <li>
+    <a href="https://downloads.yugabyte.com/releases/2.25.0.0/yugabyte-2.25.0.0-b489-el8-aarch64.tar.gz">
+      <i class="fa-brands fa-linux"></i>
+      <span>Linux ARM</span>
+    </a>
+  </li>
+</ul>
+
+**Docker:**
+
+```sh
+docker pull yugabytedb/yugabyte:2.25.0.0-b489
+```
+
+### Change log
+
+### New features
+
+#### YSQL
+
+* Added yb_make_next_ddl_statement_nonincrementing to YbDbAdminVariables for yb_db_admin role. {{<issue 23786>}} 
+* Enables creation of indexes that are covered by the main table. {{<issue 24123>}} 
+* Enables ASH by default for better monitoring and diagnostics. {{<issue 24686>}} 
+
+#### DocDB
+
+* Removes deprecated vector index storage in RocksDB. {{<issue 24854>}} 
+
+#### CDC
+
+* Enables tablet splitting on tables under CDCSDK stream by default using the GFLAG `enable_tablet_split_of_cdcsdk_streamed_tables`. {{<issue 24190>}} 
+* Speeds up tablet bootstrap under CDC by skipping initial WAL segments, controlled by `skip_wal_replay_from_beginning_with_cdc` flag. {{<issue 24516>}} 
+* Enables `kNoCacheQueryId` to prevent crashes during RocksDB operations. {{<issue 24516>}} 
+* Speeds up tablet bootstrap under CDC by fixing WAL replay. {{<issue 24516>}} 
+
+#### yugabyted
+
+* Enables better handling of complex YSQL configuration parameters using sticky connections. {{<issue 21637>}} 
+
+#### Other
+
+* Introduces a dedicated auth-backend for YSQL to handle connection authentication, enhancing security and streamlining the process. {{<issue 24147>}} 
+* Ensures correct error handling by adding a missing InvalidOid check in YSQL. {{<issue 24147>}} 
+* Enables authentication via a specialized backend, enhancing security and reliability. {{<issue 24147>}},{{<issue 24427>}} 
+* Enables toggling authentication method in YSQL using the `ysql_conn_mgr_use_auth_backend` flag. {{<issue 24147>}} 
```

Comment:
Need to put in right category and same goes for other release notes in this file.